test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/acquisitiondetails/AcquisitionDetailsControllerSpec.scala

```diff
@@ -2515,4 +2517,51 @@ class AcquisitionDetailsControllerSpec
     )
   }
 
+  def validateAcquisitionDetailsCheckYourAnswersPage(
+    acquisitionDetailsAnswers: CompleteAcquisitionDetailsAnswers,
+    doc: Document
+  ): Unit = {
+    val expectedAcquisitionMethodDisplayName = acquisitionDetailsAnswers.acquisitionMethod match {
+      case AcquisitionMethod.Bought       => messages("returns.acquisitionMethod.Bought")
+      case AcquisitionMethod.Inherited    => messages("returns.acquisitionMethod.Inherited")
+      case AcquisitionMethod.Gifted       => messages("returns.acquisitionMethod.Gifted")
+      case AcquisitionMethod.Other(value) => value
+    }
+
+    doc.select("#acquisitionMethod-answer").text() shouldBe expectedAcquisitionMethodDisplayName
+    doc.select("#acquisitionPrice-answer").text() shouldBe formatAmountOfMoneyWithPoundSign(
+      acquisitionDetailsAnswers.acquisitionPrice.inPounds()
+    )
+
+    if (acquisitionDetailsAnswers.improvementCosts === AmountInPence.zero) {
+      doc.select("#improvementCosts-answer").text shouldBe "No"
+    } else {
+      doc.select("#improvementCosts-answer").text shouldBe "Yes"
+      doc.select("#improvementCosts-value-answer").text shouldBe formatAmountOfMoneyWithPoundSign(
+        acquisitionDetailsAnswers.improvementCosts.inPounds()
+      )
+    }
+
+    if (acquisitionDetailsAnswers.acquisitionFees === AmountInPence.zero) {
+      doc.select("#acquisitionFees-answer").text shouldBe "No"
+    } else {
+      doc.select("#acquisitionFees-answer").text shouldBe "Yes"
+      doc.select("#acquisitionFees-value-answer").text shouldBe formatAmountOfMoneyWithPoundSign(
+        acquisitionDetailsAnswers.acquisitionFees.inPounds()
+      )
+    }
+
+    if (acquisitionDetailsAnswers.acquisitionDate.value.isBefore(LocalDate.of(1981, 1, 3))) {
```

Comment:
the data varies according to the asset type and the residence status. See here for the dates: https://github.com/hmrc/cgt-property-disposals-frontend/blob/master/app/uk/gov/hmrc/cgtpropertydisposalsfrontend/config/RebasingCutoffDates.scala#L25